src/utils/ExtensionUtilities.js

```diff
@@ -15,10 +17,17 @@ export function acceleratedRaycast( raycaster, intersects ) {
 		tmpInverseMatrix.copy( this.matrixWorld ).invert();
 		ray.copy( raycaster.ray ).applyMatrix4( tmpInverseMatrix );
 
+		this.getWorldScale( worldScale );
+		direction.copy( ray.direction ).multiply( worldScale );
```

Comment:
The scale axes change based on the rotation of the model so we'll want to make sure we account for the rotations when scaling here, as well. I think this should be correct:

```js
direction.copy( ray.direction ).applyMatrix4( this.matrixWorld );

// we divide by the ray.direction just in case it's non normalized
const scaleFactor = direction.length() / ray.direction.length();
```

Reply:
This still needs to be addressed, as well.